.ruby-version

```diff
@@ -0,0 +1 @@
+2.0.0-p353
```

Comment:
Can we omit this, in case someone is running a different version of ruby (we have a couple different packages of vagrant, with different ruby versions, and I think specifying this could cause problems for some people).